tsconfig.json

```diff
@@ -12,18 +12,27 @@
     "allowJs": true,
     "sourceMap": true,
     "strict": true,
+    "experimentalDecorators": true,
```

Comment:
So without this line, I get error messages complaining about instances of `@Component` in some of the `.vue` files, but those have been in the code for a year. Why is the decorator needed now?

Reply:
I'm thinking this comes from `@rancher/shell` but it is hard to explain as to why. Perhaps we can try to keep configuration for `@rancher/shell` out of our root directory and place it under some other directory like `src` or `plugins`.. this might be difficult, as it appears `@rancher/shell` has some expectations about directory structure but might be worth an attempt.